src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/WizardHeader.vue

```diff
@@ -40,5 +49,29 @@ export default defineComponent({
       },
     ],
   }),
+  computed: {
```

Comment:
We could also put this in `store/certificationType.ts` @SoLetsDev — thoughts?

Reply:
decided to leave this alone for now since there may be redesign changes to the certificationStep.